src/test/java/com/microsoft/sqlserver/jdbc/SharedTimerTest.java

```diff
@@ -5,14 +5,16 @@
 import java.util.ArrayList;
 import java.util.concurrent.*;
 
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
```

Comment:
Not part of your changes, but since we have the opportunity, please remove the * import as that's our preferred style. Also reorder as seen below. Applying our formatter will also give you this suggestion.

```
import static org.junit.jupiter.api.Assertions.assertFalse;

import java.util.ArrayList;
import java.util.concurrent.CompletableFuture;
import java.util.concurrent.ExecutionException;
import java.util.concurrent.ExecutorService;
import java.util.concurrent.Executors;
import java.util.concurrent.TimeUnit;
import java.util.concurrent.TimeoutException;

import org.junit.jupiter.api.Test;
```